golang/x/relay/types/querier.go

```diff
@@ -3,18 +3,88 @@ package types
 import (
 	"encoding/hex"
 	"fmt"
-	"strings"
 )
 
 type QueryResIsAncestor struct {
 	Digest              Hash256Digest `json:"digest"`
 	ProspectiveAncestor Hash256Digest `json:"prospectiveAncestor"`
-	IsAncestor          bool          `json:"isAncestor"`
+	Res                 bool          `json:"result"`
```

Comment:
let's add `Limit` to this struct, as the `result` may be a false-negative if the limit is low :thinking:

Reply:
Added